internal/quaiapi/api.go

```diff
@@ -1039,8 +1038,6 @@ func newRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNumber
 			GasFeeCap: (*hexutil.Big)(tx.GasFeeCap()),
 			GasTipCap: (*hexutil.Big)(tx.GasTipCap()),
 		}
-		sender := tx.ETXSender()
```

Comment:
maybe we can get rid of this ETXSender() method as well?

Reply:
The method is used in other cases, not just here